component/other/other.css

```diff
@@ -0,0 +1,14 @@
+.text-div {
+  padding: 0.5rem;
+  border: 2px solid var(--dark);
+  margin: 0.5rem 0.5rem;
```

Comment:
Maybe you can try using utility classes for properties like margin and padding.

Reply:
thanks for review , but this is my last component  if i want to add then again have to do changes all the file , i think that's ohk if using directly like